src/datasets/buildhub.md

```diff
@@ -0,0 +1,70 @@
+# Build metadata
+
+[Buildhub] is a database of metadata for official Mozilla builds of
+Firefox, Thunderbird, and Fennec (legacy Firefox for Android).
```

Comment:
Seems worth mentioning what data is available here right upfront (one of the links will only work when #621 lands).

Also probably worth mentioning buildhub support for Fenix, since it keeps on coming up.

```suggestion
Firefox, Thunderbird, and Fennec (legacy Firefox for Android).
Support for the new Firefox for Android is being tracked in 
[bug 1622948](https://bugzilla.mozilla.org/show_bug.cgi?id=1622948).

It includes data on the [build id](/concepts/terminology.md#build-id) and revision,
which can help you understand what changes went into a specific version
of the product you see in telemetry.
```